board/freescale/imx6ull_14x14_evk/imx6ull_14x14_evk.c

```diff
@@ -877,10 +886,6 @@ int board_init(void)
 	setup_gpmi_nand();
 #endif
 
-#ifdef CONFIG_VIDEO_MXS
-	setup_lcd();
```

Comment:
Where is the call to do_enable_parallel_lcd() that replaces this?

Reply:
The function pointer for do_enable_parallel_lcd is stored in the displays structure on line 826. It gets invoked in board_video_skip in arch/arm/mach-imx/video.c